<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="camera_lw">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>451</y>
      <width>256</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="log_lv">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>256</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="leftcamera_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>左相机</string>
    </property>
   </widget>
   <widget class="QLabel" name="rightcamera_label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>右相机</string>
    </property>
   </widget>
   <widget class="QLabel" name="LeftCameraLight_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="text">
     <string>左相机状态灯</string>
    </property>
   </widget>
   <widget class="QLabel" name="RightCameraLight_label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="text">
     <string>右相机状态灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftcamera_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开左边相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightcamera_btn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开右边相机</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
